axe_selenium_python/axe.py

```diff
@@ -105,7 +105,7 @@ def write_results(self, data, name=None):
         if name:
             filepath = os.path.abspath(name)
         else:
-            filepath = os.path.join(os.path.getcwd(), name)
+            filepath = os.path.join(os.path.getcwd(), "results.json")
```

Comment:
Nit, update the method's docstring to note that `results.json` will be created in the current working directory if a filename and path aren't provided

Reply:
Done